server/src/main/java/org/candlepin/async/JobBuilder.java

```diff
@@ -28,17 +28,23 @@
     private String key;
     private String name;
     private String group;
+    private Map<String, String> metadata;
     private Map<String, Object> arguments;
     private Map<String, String> restrictions;
     private int retries;
-    private boolean logJobExecution;
+    private String logLevel;
```

Comment:
I would consider `logLevel` as metadata. Is there any benefit in having it as a separate field?

Reply:
Yeah, I wavered on this a bit myself. Originally everything that was not part of the core job configuration or params was to be considered metadata; but that started to get into weird cases where only a handful of fields of the metadata map were passed through to the MDC and others were used for configuration. At that point I decided the cleanest way to differentiate the two was for anything used to configure the job or environment to be part of the main table, and everything passthrough to the MDC context to be part of the metadata.